test/e2e/config/vsphere.yaml

```diff
@@ -34,29 +34,29 @@ providers:
   - name: cluster-api
```

Comment:
I think we also have to make changes to clusterctl_upgrade_test.go (compare with: https://github.com/kubernetes-sigs/cluster-api/pull/9265/files)

Reply:
Thanks for catching it! Fixed.